cmd/performance_test/main.go

```diff
@@ -0,0 +1,154 @@
+package main
```

Comment:
do we want to have this file in this repo?

Reply:
Yes to this one as it can be used if needed.
Similar to the other main.go in the cmd/producer directory